src/experiments.js

```diff
@@ -54,6 +58,109 @@ export function isCanary(win) {
   return !!(win.AMP_CONFIG && win.AMP_CONFIG.canary);
 }
 
+/**
+ * Determines if the specified experiment is on or off for origin trials.
+ * Callers shouldcheck if the experiment is already enabled before calling this
+ * function.
```

Comment:
Since we're just using a transient experiment, we can check this internally. This should avoid parsing a token more than once.